_data/projects.yml

```diff
@@ -50,6 +50,7 @@
     visually exposes the API's structure and important patterns and serves as interactive
     API documentation.
   url: https://github.com/mulesoft/api-console
+  live_example: https://mulesoft.github.io/api-console/
```

Comment:
I was envisioning something that would allow us to define *any* call-to-action. E.g.
```
secondary_link:
  title: Live Example
  url: https://mulesoft.github.io/api-console/
```